content/english/anti-vegan/humans-are-superior.md

```diff
@@ -0,0 +1,107 @@
+---
+title: "Humans Are Superior"
+date: 2019-10-12T07:28:15+13:00
+name: tery-hardwicke
+contributors:
+    - 
+        name: tery-hardwicke
+        role: author
+    - 
+        name: sam-martin
+        role: editor
+draft: false
+description: "How to respond to the argument: We are entitled to use animals as a resource because humans are superior to nonhuman animals"
+weight: 5
+---
+# Argument
+
+> ...but, humans are superior...  
+>   
+> We are entitled to use animals as a resource because humans are superior to nonhuman animals
+
+# Rebuttal
+
+## Quick Responses
+
+> Superiority supposes an arbitrary line that falsely identifies one group as deservedly superior to another.   
+
+and  
+ 
+> Yes, humans are unique and distinct from other animals in our capacity for complex problem solving. However, pick any species and you will likely find an attribute that also makes them unique and distinct. We should celebrate difference and not deny moral consideration to those who are different.
+ 
+or 
+  
+> No unique set of abilities or characteristics, in itself, provides a criterion for ‘superiority’.  For humans to claim that humans are superior because we have specific unique attributes is just another form of prejudice. 
+ 
+or
+  
+> Even *if* it is the case we are uniquely superior, morality requires we care for those more vulnerable than ourselves; not that we treat them as a resource.
+
+besides
+
+> Do you *really* want to claim that when uniquely superior aliens discover our planet they will be morally entitled to use humans as a resource? 
```

Comment:
I think we can make this stronger and independent (I want these responses to be as discrete as possible).
Just a summary of the section below like "Imagine highly intelligent/advanced aliens discover our planet and want to use humans as a resource. Will you agree that they have the moral justification to do that?".

We should bump it further to the top too as it might not be the most analytic response of the bunch but it is one of the ones people will use the most. Let's make it easy for them to find.

Reply:
I like what Sam has written as an alternative response. I think it would be good to get the word 'superior' in there. Maybe the last sentence could be "Would you consider them to be superior and therefore (or 'in that case') have the moral justification to do that?"

Reply:
See what you think of the new version...

Reply:
Nice, easy to read and comprehend :)